dlp/system-test/inspect.test.js

```diff
@@ -885,4 +885,47 @@ describe('inspect', () => {
     }
     assert.include(output, 'INVALID_ARGUMENT');
   });
+
+  // dlp_inspect_image_file
+  it('should inspect a image file for matching infoTypes', () => {
+    const output = execSync(
+      `node inspectImageFile.js ${projectId} "resources/test.png" EMAIL_ADDRESS`
```

Comment:
Please align with Java sample where 3 info types are present in the snippet itself.